src/router/db.ts

```diff
@@ -5,14 +5,14 @@ import * as dbMan from '../services/db-manager';
 import * as iasql from '../services/iasql'
 import MetadataRepo from '../services/repositories/metadata'
 import * as telemetry from '../services/telemetry'
-import logger, { logUserErr } from '../services/logger'
+import logger from '../services/logger'
 import config from '../config'
 
 export const db = express.Router();
 
-db.get('/connect/:dbAlias', async (req, res) => {
+async function connectHandler(req: any, res: any) {
   logger.info('Calling /connect');
-  const {dbAlias} = req.params;
+  const dbAlias = req.params.dbAlias ?? req.body.dbAlias;
```

Comment:
I don't remember is `params` is always available for `req`, I think it is but just in case:
```suggestion
  const dbAlias = req.params?.dbAlias ?? req.body?.dbAlias;
```